README.md

```diff
@@ -1,7 +1,7 @@
 # junit-json-params
 
 A [Junit 5](http://junit.org/junit5/) library to provide annotations that load
-data from JSON Strings or files in parameterized tests.
+singleObject from JSON Strings or files in parameterized tests.
```

Comment:
Maybe "data as either values or deserialized into a a single object" would make more sense (If you're not a native English speaker, slap this doc into an LLM and check what makes sense there.)